CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.0.0-0] 2017-11-07
```

Comment:
This doesn't match the version in package.json. Is the intent to publish a prerelease?

Reply:
Nice catch!